packages/calypso-products/src/constants/features.ts

```diff
@@ -280,6 +280,10 @@ export const FEATURE_CUSTOMIZE_THEMES_BUTTONS_COLORS = 'customize-themes-buttons
 export const FEATURE_TRACK_VIEWS_CLICKS = 'track-views-clicks';
 export const FEATURE_COLLECT_PAYMENTS_LINK_IN_BIO = 'collect-payments-link-in-bio';
 export const FEATURE_LINK_IN_BIO_THEMES_CUSTOMIZATION = 'link-in-bio-themes-customization';
+export const FEATURE_NEWSLETTER_IMPORT_SUBSCRIBERS_FREE = 'newsletter-import-subscribers-free';
+export const FEATURE_PAYMENT_TRANSACTION_FEES_8 = 'payment-transaction-fees-8';
+export const FEATURE_PAYMENT_TRANSACTION_FEES_4 = 'payment-transaction-fees-4';
+export const FEATURE_PAYMENT_TRANSACTION_FEES_2 = 'payment-transaction-fees-2';
```

Comment:
I wonder if we should update the plans details (features) endpoint with the new features added here. I think we may not have been doing this as consistently as we should, but introducing these slugs (along with titles and descriptions) without updating the endpoint will make the two that much more out of sync. Or have these been added already and I'm just missing something?

The plans data store (that we plan to introduce and use more) produces a unified image of plans and features (it consumes and normalizes things from both `/plans` and `/plans/details` endpoints - [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/plans/resolvers.ts#L163-L177)). Ultimately, I t think this is the direction we'd want to go with the plans -> to bring things directly from the endpoints - while keeping account only of slugs and IDs on the front end, for the most part (I think?).

If we haven't been updating the endpoints, then this is something that we'll have to do at some point for everything, so it might make sense to ignore here too. So not sure. WDYT about investigating a little more to confirm (and we'll do the same for link-in-bio next).  @southp? 🤔 

some more details:

This is the API that's consumed by the plans data store: `https://public-api.wordpress.com/wpcom/v2/plans/details?locale=en`

This is the shape of the data returned from there:

<img width="500" alt="Screenshot 2023-04-22 at 7 22 12 PM" src="https://user-images.githubusercontent.com/1705499/233795833-596da968-615e-4a8e-a568-73baca2e4b58.png">

I haven't looked at the code yet. It looks like we'll need to add the feature objects to `features` (I think the ID would be the slug from here), decide on the `type`/category (might just resolve to "general"), and associate the plans with those features.

Reply:
> If we haven't been updating the endpoints, then this is something that we'll have to do at some point for everything, so it might make sense to ignore here too. So not sure. WDYT about investigating a little more to confirm (and we'll do the same for link-in-bio next

You are right. I believe there are more than a handful of cases like this that we need to confirm before moving the data into the endpoint. When we do it, I suppose we'd also have a great chances to consolidate the features data a bit better, e.g. [these shouldn't be features](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/constants/cancellation-features.ts#L4).

For now, let's keep it this way so at least it's consistent with the rest :)

Reply:
Makes sense. We'll come back to these in a more focused session.